plugins/modules/aci_l4l7_service_graph_template_term_node.py

```diff
@@ -0,0 +1,293 @@
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_owner_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        tenant=dict(type="str", aliases=["tenant_name"]),
+        service_graph=dict(type="str"),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        node_name=dict(type="str", choices=["T1", "T2"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["tenant", "service_graph", "node_name"]],
+            ["state", "present", ["tenant", "service_graph", "node_name"]],
+        ],
+    )
+
+    tenant = module.params.get("tenant")
+    service_graph = module.params.get("service_graph")
+    state = module.params.get("state")
+    node_name = module.params.get("node_name")
+
+    aci = ACIModule(module)
+
+    if node_name == "T1":
+        term_class = "vnsAbsTermNodeCon"
+        term_rn = "AbsTermNodeCon-T1"
+        term_module_object = "T1"
+        term_target_filter = {"name": "T1"}
+        name = "T1"
+    elif node_name == "T2":
+        term_class = "vnsAbsTermNodeProv"
+        term_rn = "AbsTermNodeProv-T2"
+        term_module_object = "T2"
+        term_target_filter = {"name": "T2"}
+        name = "T2"
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fvTenant",
+            aci_rn="tn-{0}".format(tenant),
+            module_object=tenant,
+            target_filter={"name": tenant},
+        ),
+        subclass_1=dict(
+            aci_class="vnsAbsGraph",
+            aci_rn="AbsGraph-{0}".format(service_graph),
+            module_object=service_graph,
+            target_filter={"name": service_graph},
+        ),
+        subclass_2=dict(
+            aci_class=term_class,
+            aci_rn=term_rn,
+            module_object=term_module_object,
+            target_filter=term_target_filter,
+        ),
+        child_classes=["vnsAbsTermConn"],
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        aci.payload(
+            aci_class=term_class,
+            class_config=dict(name=name),
+            child_configs=[
+                dict(
+                    vnsAbsTermConn=dict(
+                        attributes=dict(name="1"),
```

Comment:
why is this fixed to "1"?

Reply:
This is fixed and passed by default when we create an vnsAbsTermNodeProv or vnsAbsTermNodeCon